src/frontend/src/icons/CrateDB/CrateDB.jsx

```diff
@@ -0,0 +1,75 @@
+const SvgCrateDBIcon = (props) => (
+  <svg
+    version="1.2"
+    xmlns="http://www.w3.org/2000/svg"
+    viewBox="0 0 600 600"
+    width="600"
+    height="600"
+  >
```

Comment:
You are missing this:
```suggestion
    height="600"
    {...props}
  >
```

Reply:
Thank you. I've just amended the patch and added the missing `{...props}`.

Reply:
Hi @ogabrielluiz. I think this patch would be ready to be integrated now. Apologies for the woes around getting the logo right. May we humbly ask you to have a look at this again?

Reply:
Hi Gabriel,

do you need anything else from us to complete this patch?

With kind regards,
Andreas.

Reply:
Gentle ping. ;] ✨